pt-BR/mail/chrome/messenger/AccountWizard.dtd

```diff
@@ -67,7 +67,7 @@
      text after the "&quot;" entity in haveSmtp1.suffix, if required grammatically.
 -->
 <!ENTITY haveSmtp1.prefix "O servidor SMTP atual, “">
-<!ENTITY haveSmtp1.suffix "&quot;, will be used. You can modify outgoing server settings by choosing Account Settings from the Tools menu.">
+<!ENTITY haveSmtp1.suffix "&quot;, será utilizado. Você pode alterar as configurações do servidor de saída selecionando a opção Configurações de conta no menu Ferramentas.">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY haveSmtp1.suffix "&quot;, será utilizado. Você pode alterar as configurações do servidor de saída selecionando a opção Configurações de Conta no menu Ferramentas.">